spec/lib/models/concerns/audit_trail_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "spec_helper"
+
+module Timelines
+  describe AuditTrail, type: :model do
+    let(:mock_resource) { described_class.new }
+    let(:mock_events) {
+      [
+        {
+          name: "mock event 1"
+        },
+        {
+          name: "mock event 2"
+        }
+      ]
+    }
+
+    before { allow(mock_resource).to receive(:events).and_return(mock_events) }
+
+    it "returns sorted events" do
+      expect(mock_resource.events).to eq(mock_events)
+    end
+
+    describe "when reverse is true" do
+      let(:dummy_class) { Class.new { extend ::Timelines::HasAuditTrail } }
+
+      it "returns events in reverse order if reverse is true" do
+        mock_resource.reverse = true
```

Comment:
@Craggar sorry to be a pain but I'm struggling to set the `reverse` attribute here. I tried the above code, as well as `.update_attributes(reverse: false)`, and creating a new mock class with `let(:mock_resource) { described_class.new(reverse: true, events: events }` and haven't managed to get this working. Any ideas?